doc/howto/instances_routed_nic_vm.md

```diff
@@ -0,0 +1,42 @@
+(instances-routed-nic-vm)=
+# How to add a routed NIC device to a virtual machine
+
+When adding a [routed NIC device](../reference/devices_nic.md#nictype-routed) to an instance, the instance must be configured to use the link-local gateway IPs as default routes. 
+For containers, this is configured for you automatically.
+For virtual machines, the gateways must be configured manually or via a mechanism like `cloud-init`.
+
+To configure the gateways with `cloud-init`, firstly initialize an instance:
+
+    lxc init ubuntu:jammy jammy --vm
+
+Then add the routed nic device:
+
+    lxc config device add jammy eth0 nic nictype=routed parent=my-parent-network ipv4.address=192.0.2.2 ipv6.address=2001:db8::2
+
+Where `my-parent-network` is your parent network, and the IPv4 and IPv6 addresses are within the subnet of the parent.
+
+Then set the following config on the instance:
+
+    lxc config set jammy cloud-init.network-config "$(cat <<EOF
+    network:
+      version: 2
+      ethernets:
+        enp5s0:
+          link-local:
+          - ipv4
+          - ipv6
+          routes:
+          - to: default
+            via: 169.254.0.1
+          - to: default
+            via: fe80::1
+          addresses:
+          - 192.0.2.2/32
+          - 2001:db8::2/128
```

Comment:
I guess I should mention that this is netplan configuration and link to their documentation?

Reply:
Yes, I would add that before the command. Something like:
```
Then add the netplan configuration to `cloud-init`. See the [netplan documentation](...) for more information.
For example, use the following command:
```